lib/requireContent.js

```diff
@@ -0,0 +1,15 @@
+(function() {
+  var global = this;
+  require.load = function (context, moduleName, url) {
+    var xhr;
+    xhr = new XMLHttpRequest();
+    xhr.open("GET", chrome.extension.getURL(url) + '?r=' + new Date().getTime(), true);
+    xhr.onreadystatechange = function (e) {
+      if (xhr.readyState === 4 && xhr.status === 200) {
+        eval.call(global, xhr.responseText + '\n//@ sourceURL=' + url);
+        context.completeLoad(moduleName)
+      }
+    };
+    xhr.send(null);
+  };
+})();
```

Comment:
Er, I copied this from somewhere sorry... Sam C and I were googling around how to add require JS to a chrome extension, it seemed that something along these lines was necessary but I didn't quite figure out why :/ 

Found it - https://prezi.com/rodnyr5awftr/requirejs-in-chrome-extensions/